Sources/Model/Localized.yml

```diff
@@ -417,80 +501,96 @@ noSelectionDescription:
     en: Select a set in the sidebar
     de: Wähle ein Set in der Seitenleiste aus
     it: Seleziona un set nella barra laterale
+    pt_BR: Selecione um set na barra lateral
 
 noSets:
     en: No Sets
     de: Keine Sets verfügbar
     it: Nessun Sets
+    pt_BR: Nenhum Set
 
 noSetsDescription:
-    en: Create a set to start practicing
-    de: Erstelle ein Set, um mit dem Lernen zu beginnen
+    en: Create a new set using the <b>+</b> button in the sidebar
+    de: Erstelle mit dem <b>+</b>-Knopf in der Seitenleiste ein neues Set
+    it: Crea un nuovo set utilizzando il pulsante <b>+</b> nella barra laterale
+    pt_BR: Crie um novo set utilizando o botão <b>+</b> na barra lateral
 
 overview:
     en: Overview
     de: Übersicht
     it: Panoramica
+    pt_BR: Visão Geral
 
 studySwitcher:
     en: Study
     de: Lernen
     it: Studio
+    pt_BR: Estudar
 
 test:
     en: Test
     de: Testen
     it: Test
+    pt_BR: Teste
 
 newSet:
     en: New Set
     de: Neues Set
     it: Nuovo Set
+    pt_BR: Novo Set
 
 question(index):
     en: Question (index)
     de: Frage (index)
     it: Domanda (index)
+    pt_BR: Questão (index)
 
 quizlet:
     en: Quizlet
     de: Quizlet
     it: Quizlet
+    pt_BR: Quizlet
 
 anki:
     en: Anki
     de: Anki
     it: Anki
+    pt_BR: Anki
 
 showTutorial:
     en: Show Tutorial
     de: Tutorial zeigen
     it: Mostra tutorial
+    pt_BR: Mostrar Tutorial
 
 comma:
     en: Comma
     de: Komma
     it: Virgola
+    pt_BR: Vírgula
 
 tab:
     en: Tab
     de: Tab
     it: Scheda
+    pt_BR: Aba
 
 newLine:
     en: New Line
     de: Neue Zeile
     it: Nuova riga
+    pt_BR: Nova Linha
 
 semicolon:
     en: Semicolon
     de: Semikolon
     it: Punto e virgola
-    
+    pt_BR: Ponto e vírgula
+
 editSetDescription:
```

Comment:
This string has been added in recent commits. It would be nice if you could translate this as well (not necessary for merging, but still).